source/Renderer/shaders/pbr.frag

```diff
@@ -242,7 +242,7 @@ MaterialInfo getSheenInfo(MaterialInfo info)
     info.sheenRoughnessFactor = u_SheenRoughnessFactor;
 
     #ifdef HAS_SHEEN_COLOR_MAP
-        vec4 sheenColorSample = texture(u_SheenColorSampler, getSheenColorUV());
+        vec4 sheenColorSample = sRGBToLinear(texture(u_SheenColorSampler, getSheenColorUV()));
```

Comment:
we should be setting the correct format on JS side instead of converting in the shader